fem/lininteg.cpp

```diff
@@ -351,6 +351,36 @@ void VectorFEDomainLFIntegrator::AssembleDeltaElementVect(
 }
 
 
+void VectorFEBoundaryLFIntegrator::AssembleRHSElementVect(
+   const FiniteElement &el, ElementTransformation &Tr, Vector &elvect)
+{
+   int dof = el.GetDof();
+
+   shape.SetSize(dof);
+   elvect.SetSize(dof);
+   elvect = 0.0;
+
+   const IntegrationRule *ir = IntRule;
+   if (ir == NULL)
+   {
+      int intorder = 2*el.GetOrder();  // <----------
```

Comment:
For this integrator, it should be sufficient to use `intorder = el.GetOrder()` and even `intorder = el.GetOrder()-1` for the standard RT spaces.